tests/test_openjdk_devel.py

```diff
@@ -86,6 +87,10 @@ def test_maven_present(auto_container):
     assert auto_container.connection.run_expect([0], "mvn --version")
 
 
+@pytest.mark.xfail(
```

Comment:
This is good but not enough. See https://openqa.suse.de/tests/16039106, we need to softfail also the following test runs:

* `test_glibc_present`
* `test_product`
* `test_bci_eula_is_correctly_available`
* `test_java_home`
* `test_maven_present`

All of them fail due to the "can only create exec sessions on running containers: container state improper" error because the container cannot be started.

Reply:
okay, we can fix that by setting a custom entry point in the tests